dashboard/app/lib/javascripts/dashboard/main.js

```diff
@@ -65,6 +66,24 @@ window.Dashboard = {
 		Marbles.history.navigate('login'+ redirectPath);
 	},
 
+	__isCertInstalled: function() {
+		var self = this, dispatchIfCertIsMissing = function(a, b) {
+			if (a + b !== 200 + 200) {
```

Comment:
The logic here need to be `a === 200 && b === 0` (`http` ping succeeded so we know the controller is up, `https` failed so we know the cert isn't installed/working).